release_helper/issue_management/linear/__init__.py

```diff
@@ -13,18 +17,31 @@
 )
 
 
+class CustomClient(GeneratedClient):
+    def __init__(
+        self,
+        url: str = "",
+        headers: dict[str, str] | None = None,
+        http_client: httpx.Client | None = None,
+    ):
+        super().__init__(url, headers, http_client)
+        self.http_client = httpx.Client(headers=self.headers, timeout=httpx.Timeout(30))
+
+
 class IssueManagementLinear:
     def __init__(self):
         self.client = self.get_client()
 
     @staticmethod
-    def get_client() -> Client:
+    def get_client() -> CustomClient:
         token = os.environ.get("HELPER_LINEAR_TOKEN")
         headers = {
             "Content-Type": "application/json",
             "Authorization": f"{token}",
         }
-        linear_client = Client(url="https://api.linear.app/graphql", headers=headers)
+        linear_client = CustomClient(
+            url="https://api.linear.app/graphql", headers=headers
+        )
```

Comment:
Instead of making an entirely new CustomClient, can you not just add the new httpx client to the init?

```
linear_client = Client(
    url="https://api.linear.app/graphql",
    headers=headers,
    http_client=httpx.Client(headers=self.headers, timeout=httpx.Timeout(30))
)
```

Seems like the one that they generated already has the provision there built-in.